src/scripts/main.js

```diff
@@ -1 +1,44 @@
-console.log("hey");
+$(function() {
+  // On page load / resize, force now & next header boxes to be equal (sqaure) size.
+  var $el = $('.current-next-live');
+  var $el2 = $('.current-next-now');
+  var $el3 = $('.current-next-next');
+  var $window = $(window).on('resize', function() {
+    var width = $el.outerWidth();
+    $el.height(width);
+    $el2.height(width);
+    $el3.height(width);
+  }).trigger('resize'); //on page load
+});
+
+
+//Used for autoupdating the homepage Now & Next.
+function updateShow() {
```

Comment:
I don't know the API works to be honest, it worked currently, so I just copied it. If you feel we can do it another way, then sure.